pipeline/Makefile

```diff
@@ -0,0 +1,109 @@
+.DEFAULT_GOAL := help
+
+CONFIG_PATH := brca_pipeline_cfg.mk
+include $(CONFIG_PATH)
+
+checkout: ## checking out code base
+	[ -d $(CODE_BASE) ] || git clone $(BRCA_GIT_REPO) $(CODE_BASE)
+	cd $(CODE_BASE) && git checkout $(GIT_COMMIT)
+
+build-docker: ## build main pipeline docker image
+	cd $(CODE_BASE)/pipeline/docker && ./build_docker_image.sh $(GIT_COMMIT) $(RELEASE_TAG) || { echo "$(date) Something wrong with docker image creation"; exit 1; }
+
+download-resources: ## download resources files
+	$(CODE_BASE)/pipeline/download_resources_files.sh $(RESOURCES_DIR) >> "$(LOG_DIR)/download_resources_$(RELEASE_TAG).log" 2>&1 || { echo "Downloading resource files failed"; exit 1; }
```

Comment:
`download_resource_files.sh` removes the .gz files when they're unzipped, which causes them to be re-downloaded with each run. I realize this is a problem with that script and not this PR per se, but it's something to think about.

Reply:
yes, good point! will fix it.